packages/suite/src/views/suite/SwitchDevice/DeviceItem/WalletInstance.tsx

```diff
@@ -152,7 +156,7 @@ export const WalletInstance = ({
                         )}
                         {instance.state ? (
                             <MetadataLabeling
-                                defaultVisibleValue={<WalletLabeling device={instance} />}
+                                defaultVisibleValue={walletLabel ?? defaultWalletLabel}
```

Comment:
`WalletLabeling` component is heavy component that handles prefix of the device. In this case is is not needed. 

It uses `defaultWalletLabel` under the hood to get the default value.